pullrequests-list.txt

```diff
@@ -1 +1 @@
-
+https://github.com/asaulyak/a-team-advancedjs/pull/5

```

Comment:
remove this file
it supposed to be excluded

Reply:
deleted